textile.go

```diff
@@ -274,7 +275,15 @@ func (x *migrateCmd) Execute(args []string) error {
 }
 
 func (x *daemonCmd) Execute(args []string) error {
-	if err := buildNode(x.PinCode, x.RepoPath); err != nil {
+	logLevels := make(map[string]string)
+	for _, log := range x.Logs {
+		split := strings.SplitN(log, ":", 2)
```

Comment:
if the syntax isn't right, it'll be silently ignored... this seems appropriate? could be convinced to add validation..?

Reply:
feels right, yeah